### PR TITLE
fix(module:slider): fix range slider not working for arrow keys

### LINKED ISSUE #6586: Using range slider with arrow keys doesn't work
### Reproduction link
[https://stackblitz.com/edit/angular-c3rff8?file=src/app/app.component.ts](https://stackblitz.com/edit/angular-c3rff8?file=src/app/app.component.ts)

### Steps to reproduce
There are 2 situations that the slider doesn't work like it's supposed to:

First:
1. Click on the 'click me' text, this is just a \<p\>
1. Press 'tab' so you are on the first handle of the slider
1. Try to move it with your arrow keys

Second: 
1. Click the first handle and move it around a bit with your mouse
1. While the first handle is still in focus press 'tab' once so you're on the second handle
1. Try to move the second handle with your arrow keys

### What is expected?
First situation:
The first handle moves to the left when you press 'down' or 'left' and to the right when you press 'up' or 'right'.

Second situation:
The second handle moves to the left when you press 'down' or 'left' and to the right when you press 'up' or 'right'.

### What is actually happening?
First situation:
The handle doesn't move when pressing the arrow keys.

Second situation:
The first handle moves instead of the second.

| Environment | Info |
|---|---|
| ng-zorro-antd | 11.4.0 |
| Browser | Chrome |

---
In the second situation the handle that got moved the last with the mouse will be the one that moves when you press the arrow keys. Even though you might have tabbed to the other one.

### PR DESCRIPTION
**Pull request by stygian-desolator**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application (the showcase website) / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

close #6586 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by wenqi73** [comment]
@wendellhu95 Please review this.

File: components/slider/slider.component.ts [hunk 1]
```diff
@@ -103,6 +103,7 @@ import { NzExtendedMark, NzMarks, NzSliderHandler, NzSliderShowTooltip, NzSlider
         [tooltipVisible]="nzTooltipVisible"
         [tooltipPlacement]="nzTooltipPlacement"
         [dir]="dir"
+        (keydown)="onHandleKeyDown(handleIndex)"
```
Comment (hsuanxyz) [inline components/slider/slider.component.ts:106]:
```suggestion
        (focusin)="onHandleKeyDown(handleIndex)"
```
Is it possible to use focusin here?

Comment (stygian-desolator) [reply components/slider/slider.component.ts]: updated, pls review it again~

**Review by wendellhu95 — APPROVED** [review]
LGTM. Thanks!

**Review by hsuanxyz — APPROVED** [review]

**Pull request merged**